app/models/grda_warehouse/hmis_form.rb

```diff
@@ -271,23 +273,22 @@ def self.set_part_of_a_family
 
       # anyone with the most-recent score > 2 is included
       if score[:score].present? && score[:score] > 2
-        family_source_client_ids << client_id
+        family_destination_client_ids << client_id
         next
       end
 
       # anyone who is under 60 and has a score of 1 in the most-recent vi-spdat
-      client = potential_clients[client_id]
+      client = potential_destination_clients[client_id]
       next unless client
 
       collected_at = score[:collected_at]
       age = client.age_on(collected_at.to_date)
       next if age.blank? || age >= 60
```

Comment:
Should this have guard of `score[:score] == 1`? Or is 2 also OK?

Reply:
You'd never get here with a 2, those are next'd on 275.

Reply:
Yes, 2 should be ok, only that they have a score.

Reply:
Maybe either change 275 to be `>= 2`, or adjust the comment?